roles/branch-validation/tasks/main.yml

```diff
@@ -92,6 +82,17 @@
     delay: 5
     timeout: 300
     host: "{{ fip.floating_ip.floating_ip_address }}"
+  when: not local_wait_for
+
+- name: wait for instances to be reachable
+  sudo: no
```

Comment:
Lets change `sudo` to `become` in this role. It'll help with ansible 2.0 moving.